here/module.go

```diff
@@ -5,11 +5,11 @@ import (
 )
 
 type Module struct {
-	Path      string `json:"path"`
-	Main      bool   `json:"main"`
-	Dir       string `json:"dir"`
-	GoMod     string `json:"go_mod"`
-	GoVersion string `json:"go_version"`
+	Path      string
```

Comment:
Why were the json tags removed?

Reply:
the json output is:
```json
 "Module": {
                "Path": "github.com/friendsofgo/sample-pkgr",
                "Main": true,
                "Dir": "/Users/aperez/dev/go/src/github.com/friendsofgo/pkgr",
                "GoMod": "/Users/aperez/dev/go/src/github.com/friendsofgo/pkgr/go.mod",
                "GoVersion": "1.13"
        }
```
so the json tags are not necessary in this case